SharpSnmpLib/ByteTool.cs

```diff
@@ -127,26 +148,75 @@ public static string Convert(byte[] buffer)
             return BitConverter.ToString(buffer).Replace('-', ' ');
         }
 
-        internal static byte[] ParseItems(params ISnmpData[] items)
+        internal static byte[] ParseItems(ISnmpData item1, ISnmpData item2, ISnmpData item3, ISnmpData item4)
```

Comment:
Is is always the case that we get 4 items here?

Reply:
Yeah, I just removed params and checked how many we were inserting